src/screens/claimables/ClaimingSponsoredClaimable.tsx

```diff
@@ -86,6 +90,10 @@ export const ClaimingSponsoredClaimable = ({ claimable }: { claimable: Sponsored
         );
       }
     },
+    onSettled: () => {
+      // Clear and refresh claimables data 20s after claim button is pressed, regardless of success or failure
+      delay(() => queryClient.invalidateQueries(queryKey), 20_000);
```

Comment:
is `delay` the same as `setTimeout`? if so can we stop using lodash?

Reply:
why?